Misc/NEWS.d/next/Core and Builtins/2022-04-05-11-29-21.bpo-47212.leF4pz.rst

```diff
@@ -0,0 +1,4 @@
+- raise IndentationError instead of SyntaxError for a bare except with no following indent
+- improve SyntaxError locations for un-parenthesized generator used as arguments
+- re-generate the parser
+- add tests for the above two fixes
```

Comment:
Should fix the Docs CI failure :)

```suggestion
Raise :exc:`IndentationError` instead of :exc:`SyntaxError` for a bare
``except`` with no following indent. Improve :exc:`SyntaxError` locations for
an un-parenthesized generator used as arguments. Patch by Matthieu Dartiailh.
```

Reply:
Thanks